sh/filter_bgen.sh

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/bash
 
-source "/homes/nber/mahdimir/.zshrc"
+source "/homes/nber/mahdimir/.zshrc" # I generally recommend against using personal space or settings in project areas
```

Comment:
There can be exceptions, but keeping personal settings out of it can make it easier for work to be passed around if needed.  You can always import whatever settings you need here, or at least have a project-specific import file that you can source.